users/stdiscosrv.rst

```diff
@@ -70,7 +64,7 @@ By default, Syncthing uses a number of global discovery servers, signified by
 the entry ``default`` in the list of discovery servers. To make Syncthing use
 your own instance of stdiscosrv, open up Syncthing's web GUI. Go to settings,
 Global Discovery Server and add stdiscosrv's host address to the comma-separated
-list, e.g. ``https://disco.example.com:8443/v2/``. Note that stdiscosrv uses port
+list, e.g. ``https://disco.example.com:8443/``. Note that stdiscosrv uses port
```

Comment:
Why is /v2 removed?

Reply:
It's not necessary any more with either the existing global servers or just stdiscosrv by it's own, as there is no longer any v1 support. It'll allow /v2 if it comes in, though.